### DISCUSSION TITLE
KDE plasma package:fork or integration

### DISCUSSION THREAD

**Discussion by telmobss1** [lxqt/lxqt#1943] [Ideas]
As a newcomer here, I would like to ask all the developers, if you think the lxqt project should fork the KDE plasma packages and modify them to be part of the lxqt packages,Try to integrate without possibility for forks

**Comment by tsujan** [comment]
LXQt is a *Qt-based* DE, not a fork of KDE.  KDE Plasma isn't related to LXQt.

**Reply by telmobss1** [reply]
I changed the title because I verified that I did not transmit my question correctly

Most Qt applications are part of the KDE ecosystem, such as Discover, kwin and most applications starting with K

A minha pergunta era,vocês estão em pensar em criar uma alternativa ao Discover usando Qt para o lxqt?

**Reply by tsujan** [reply]
> Most Qt applications are part of the KDE ecosystem

It seems you're new to Qt and have mistaken it for a KDE library.

 1. KDE is a desktop environment that was started based on an early version of Qt a long time ago (when there were problems in Qt's license).
 2. LXQt is a totally different desktop environment based on Qt and GLib. It started when Qt's version was 4.
 3. Qt is an open-source and versatile toolkit, which can be used for writing apps that are independent of all desktop environments and usable everywhere. There are many Qt apps that are independent of KDE and many developers write programs in Qt exactly because it covers most platforms.
 4. GLib is the library LXQt uses for file and mime-type management. It's made and developed by the Gnome team.

**Comment by wxl** [comment]
KDE libraries are used in LXQt, just as GNOME libraries are used in LXDE. However, LXQt doesn't include KDE, just as LXDE doesn't include GNOME. There's no reason to integrate these as they're libraries that can be used by a variety of software projects. They're generally applicable, in other words. We don't need to take on the responsibility of maintaining those all ourselves. Forking is something one does as a means of last resort when the upstream developer fails to accommodate your needs and you don't want to create an alternative solution yourself from scratch. We don't have such a problem.

**Comment by antis81** [comment]
@telmobss1 It is essential to understand that LXQt does not create packages (nor do we define what contents they have)!

So who creates the packages? This is the job (and self-understanding) of distributions! For Lubuntu that means, it has to come in "flavors": Lubuntu+openbox, Lubuntu+kwin/kwinft, etc.
(The better naming would be: Ubuntu+LXQt+openbox/kwin.)

The [lxqt-kcm-integration](https://github.com/lxqt/lxqt-kcm-integration) repository may help you getting a basic understanding with what is going on and where you can contribute.
